stacks/stacks-v2.yaml

```diff
@@ -303,7 +303,7 @@ stacks:
       - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/main/stacks/trino-superset-s3/s3-connection.yaml
       - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/main/stacks/trino-superset-s3/hive-metastore.yaml
       - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/main/stacks/trino-superset-s3/trino.yaml
-      - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/main/stacks/trino-superset-s3/superset.yaml
+      - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/feat/marispace-demo/stacks/trino-superset-s3/superset.yaml
```

Comment:
Reminder to change back

Reply:
https://github.com/stackabletech/demos/pull/88/commits/2ece0adffa9f874b5eed5654827c1d478bab356b